Previously, there was a circular dependency between testtools and fixtures.
Upstream mostly fixed that - this patch is the last thing necessary to
finish the job.

From f08b45932ad4f213d26be7ebc2658678cc213bae Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@canonical.com>
Date: Wed, 1 Jun 2022 16:13:07 +0100
Subject: [PATCH] Remove testtools from requirements.txt as well

--- fixtures-4.0.0/requirements.txt
+++ fixtures-4.0.0/requirements.txt
@@ -1,2 +1 @@
 pbr>=5.7.0
-testtools>=2.5.0
